spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
OK will do.  

re: creating a new SMS message, I was imagining that but was trying to keep this PR small(ish).. but I think you may be right i probably makes sense to make it part of this PR.